src/libs/actions/User.js

```diff
@@ -362,15 +400,60 @@ function subscribeToExpensifyCardUpdates() {
  * @param {String} skinTone
  */
 function setPreferredSkinTone(skinTone) {
-    NameValuePair.set(CONST.NVP.PREFERRED_EMOJI_SKIN_TONE, skinTone, ONYXKEYS.PREFERRED_EMOJI_SKIN_TONE);
+    const optimisticData = [
+        {
+            onyxMethod: CONST.ONYX.METHOD.SET,
+            key: ONYXKEYS.PREFERRED_EMOJI_SKIN_TONE,
+            value: skinTone,
+        },
+    ];
+    API.write('SetPreferredEmojiSkinTone', {
+        value: skinTone,
+    }, {optimisticData});
 }
 
 /**
  * Sync frequentlyUsedEmojis with Onyx and Server
  * @param {Object[]} frequentlyUsedEmojis
  */
 function setFrequentlyUsedEmojis(frequentlyUsedEmojis) {
```

Comment:
```suggestion
function updateFrequentlyUsedEmojis(frequentlyUsedEmojis) {
```

Similar to Marc's suggestion below.